values-schema.yaml

```diff
@@ -172,8 +172,14 @@ definitions:
     pattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
     type: string
   env:
-    pattern: '^[a-zA-Z0-9_]*$'
-    type: string
+    items:
+      additionalProperties: false
+      patternProperties:
+        '^[a-zA-Z0-9_]*$':
```

Comment:
env variable name should not start from number as these are resaved names: e.g.: $0, $3
Please use the following regex:
```[a-zA-Z_]{1,}[a-zA-Z0-9_]{0,}```

Reply:
I don't see any response or code change. I do not understand why it is marked as resolved